.github/workflows/CI.yml

```diff
@@ -0,0 +1,65 @@
+name: Run CI
+on:
+  # Triggers the workflow on push events but only for the "main" branch
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+    inputs:
+      liquid_sdk_ref:
+        description: 'Liquid SDK commit/tag/branch reference'
+        required: false
+        type: string
+        default: 'main'
+
+jobs:
+  build:
+    runs-on: macOS-latest
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: 🏗️ Setup l-breez repository
+        uses: actions/checkout@v4
+        with: 
+          path: 'lbreez'
+
+      # TODO: Liquid - Revert once breez-sdk dependency is removed
+      - name: 🏗️ Setup breez-sdk repository
+        uses: actions/checkout@v4
+        with:
+          repository: 'breez/breez-sdk'
+          ssh-key: ${{secrets.REPO_SSH_KEY}}
+          path: 'breez-sdk'
+          ref: 'fca80043d011f213741ddae93973ed264a8b6ddb'
```

Comment:
Would it be worth adding this as a `breez_sdk_ref` input? Maybe a specific branch would be better

Reply:
Changed it target `flutter_rust_bridge_v2` by default but this change will only be reflected once workflow file is merged to `main`.